examples/seismic/viscoacoustic/operators.py

```diff
@@ -93,14 +170,93 @@ def ren(model, geometry, v, p, **kwargs):
     pde_v = v - s * b * grad(p)
     u_v = Eq(v.forward, damp * pde_v)
 
-    pde_u = p - s * vp * vp * rho * div(v.forward) + \
-        s * ((vp * vp * rho) / (w * qp)) * div(b * grad(p))
-    u_p = Eq(p.forward, damp * pde_u)
+    pde_p = p - s * vp * vp * rho * div(v.forward) + \
+        s * ((vp * vp * rho) / (w0 * qp)) * div(b * grad(p))
+    u_p = Eq(p.forward, damp * pde_p)
 
     return [u_v, u_p]
 
 
-def deng_mcmechan(model, geometry, v, p, **kwargs):
+def ren_2nd_stencil(model, geometry, v, p, **kwargs):
```

Comment:
Can't this and the above be combined and have a `'case'` `kwarg`?

Reply:
We understand this approach but based on the available examples we choose to make this case option on equation methods, before calling these methods.